tmux.conf

```diff
@@ -27,6 +27,12 @@ set -g status-fg '#aaaaaa'
 set -g status-left ''
 set -g status-right ''
 
+# allow ctrl + b + - to split window horizontally
+bind - split-window -v 
```

Comment:
A small change here will open the new pane at the same path as the current pane:

```
bind-key - split-window -v  -c '#{pane_current_path}'
```

Reply:
Makes sense. I updated the bindings. Thanks